frontend/src/metabase/lib/query_time.js

```diff
@@ -66,6 +67,16 @@ export function computeFilterTimeRange(filter) {
     end = absolute(values[1]).endOf(bucketing);
   }
 
+  if (values[0][2] === "week") {
+    const startOfWeek = MetabaseSettings.get("start-of-week");
+    const startOfWeekIndex = moment()
+      .localeData()
+      .weekdays()
```

Comment:
If Metabase is set to a language other than English, looks like this may be problematic.

Consider an array of weekdays in Italian.

<img width="666" alt="image" src="https://user-images.githubusercontent.com/380816/180013766-f0ae44fa-de4d-4aad-898d-fd17c32e71c5.png">